proprietary/tokens/src/components/ams/dialog.tokens.json

```diff
@@ -5,16 +5,16 @@
       "border": { "value": "0" },
       "max-inline-size": { "value": "min(87.69vw, 45rem)" },
```

Comment:
This is a bit of an odd number, is it based on anything?

Reply:
I don’t remember. Hasn’t changed in this PR – I added [a ticket](https://gemeente-amsterdam.atlassian.net/browse/DES-832) for it.